src/app/app.component.css

```diff
@@ -0,0 +1,29 @@
+div.side-bar {
+  background-color: gray;
+  height: 100vh;
+}
+
+div.name {
+  padding-top: 15px;
+  text-align: center;
+  color: black;
+  font-size: 30pt;
+  font-weight: bold;
+  font-family: Garamond;
+}
+
+div.work-experience {
+  padding-top: 70px;
+  text-align: center;
+  color: white;
+  font-size: 14pt;
+  font-family: Futura, Trajan;
+}
+
+div.education {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the name of this overqualified element. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Aoverqualified-elements)